worlds/tits_the_3rd/names/region_name.py

```diff
@@ -46,3 +46,31 @@ class RegionName:
     chapter_7_end = "Chapter 7 End"
     sky_bandit_fight = "Sky Bandit Fight"
     star_door_10 = "Star Door 10"
+
+    kevin = "Kevin"
+    kevin_chapter1 = "Kevin - Chapter 1"
```

Comment:
I don't see any benefit to making chapter regions for each character. Furthermore, I this doesn't make sense if we, say, allow the player to access a later chapter early (meaning they might have access to chapter 5 and can get levels before chapter 2).

Why not keep the regions as "level X", and put having the character as an item requirement to check the item location?